<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="1" rowspan="4">
     <widget class="QLabel" name="capturePicture">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>50</horstretch>
        <verstretch>100</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSlider" name="horizontalSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>камера №</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="buttonCaptureStart">
           <property name="text">
            <string>запустити та перевірити камеру</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="processingImageShow">
      <property name="text">
       <string>виводи  зображення, що обробляється</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QFrame" name="frame">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QPushButton" name="buttonAnalizStop">
         <property name="text">
          <string>зупинити аналіз</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="buttonAnalizStart">
         <property name="text">
          <string>розпочати аналіз</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Інформація</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionTo_fail"/>
    <addaction name="actionTo_graphik"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionShowFile"/>
   </widget>
   <widget class="QMenu" name="menuCapture">
    <property name="title">
     <string>Capture</string>
    </property>
    <widget class="QMenu" name="menuChoise_capture">
     <property name="title">
      <string>Список підкючених камер</string>
     </property>
     <addaction name="action1"/>
    </widget>
    <addaction name="menuChoise_capture"/>
    <addaction name="separator"/>
    <addaction name="actionDisable_capture"/>
    <addaction name="separator"/>
    <addaction name="actionHide_capture"/>
    <addaction name="actionShow_capture"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionConfiguration_of_program"/>
    <addaction name="actionConfiguration_of_capture"/>
    <addaction name="actionOf_export"/>
    <addaction name="separator"/>
    <addaction name="actionStart_testing"/>
   </widget>
   <widget class="QMenu" name="menuInformation">
    <property name="title">
     <string>information</string>
    </property>
    <addaction name="actionHelper"/>
    <addaction name="actionInformation"/>
   </widget>
   <addaction name="menuCapture"/>
   <addaction name="menuExport"/>
   <addaction name="menuSetting"/>
   <addaction name="menuInformation"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHelper">
   <property name="text">
    <string>Helper</string>
   </property>
  </action>
  <action name="actionDisable_capture">
   <property name="text">
    <string>Disable capture </string>
   </property>
  </action>
  <action name="actionConfiguration_of_program">
   <property name="text">
    <string>of program...</string>
   </property>
  </action>
  <action name="actionConfiguration_of_capture">
   <property name="text">
    <string>of capture... </string>
   </property>
  </action>
  <action name="actionOf_export">
   <property name="text">
    <string>of export</string>
   </property>
  </action>
  <action name="actionStart_testing">
   <property name="text">
    <string>start testing..</string>
   </property>
  </action>
  <action name="actionTo_fail">
   <property name="text">
    <string>to fail</string>
   </property>
  </action>
  <action name="actionTo_graphik">
   <property name="text">
    <string>to graphik</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>print </string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="text">
    <string>information</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="actionHide_capture">
   <property name="text">
    <string>Hide capture</string>
   </property>
  </action>
  <action name="actionShow_capture">
   <property name="text">
    <string>Show capture</string>
   </property>
  </action>
  <action name="actionShowFile">
   <property name="text">
    <string>показати файл</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
